packages/authhero/src/components/Trans.tsx

```
@@ -0,0 +1,40 @@
+
+const Trans: FC<TransProps> = (params) => {
+  const { i18nKey, values, components } = params;
+
+  const translation = i18next.t(i18nKey, values);
+  const regex = /<(\d+)>(.*?)<\/\d+>/g;
+
+  const result: (string | JSXNode)[] = [];
+  let lastIndex = 0;
+  let match: RegExpExecArray | null;
+
+  while ((match = regex.exec(translation)) !== null) {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)